features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/controllers/login.jag

```diff
@@ -1,36 +1,38 @@
 <%
 (function(){
+    include("/extensions/webapp/modules/jagg/jagg.jag");
+    var ssoIdpProvider = jagg.module('manager').getAPIProviderObj();
     var dataConfi = require('/config/publisher.js').config(),
-            sso = require("sso"),
-            process = require("process"),
-            sessionId = session.getId(),
-            requestURI,
-            requestedPage = request.getParameter("requestedPage"),
-            relayState = '/publisher',
-            encodedSAMLAuthRequest = sso.client.getEncodedSAMLAuthRequest(dataConfi.ssoConfiguration.issuer),
-            serverHost = process.getProperty('server.host'),
-            log = new Log(),
-            postUrl = dataConfi.ssoConfiguration.identityProviderURL;
+        sso = require("sso"),
+        process = require("process"),
+        sessionId = session.getId(),
+        requestURI,
+        requestedPage = request.getParameter("requestedPage"),
+        relayState = '/publisher',
+        encodedSAMLAuthRequest = sso.client.getEncodedSAMLAuthRequest(dataConfi.ssoConfiguration.issuer),
+        serverHost = process.getProperty('server.host'),
+        log = new Log(),
+        postUrl = ssoIdpProvider.getIdentityProviderUrl();
 
     if (!session.get("Loged")) {
```

Comment:
The issue was there in the original code but not with this PR. There are 6 code occurrences like this. I will fix those in all relevant locations.